index.html

```diff
@@ -0,0 +1,321 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.css">
+    </link>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Serif:opsz@8..144&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+
+<body>
+    <nav class="navbar">
+        <header><a class="header" href="#" >John Doe </a></header>
+      <div class="navbarmenu">
+          <div class="btn-line"></div>
+          <div class="btn-line"></div>
+          <div class="btn-line"></div>
+      </div>
+        <ul class="itemslist" id="elements">
+            <li class="listitem"><a href="#portfolio">Portfolio</a></li>
+            <hr class="spaceitem" />
+            <li class="listitem" ><a href="#about">About</a></li>
+            <hr class="spaceitem"/>
+            <li class="listitem" ><a href="#contact">Contact</a></li>
+            <hr class="spaceitem"/>
```

Comment:
- [ ] Great job. please ensure the menu links take you to the correct page sections. the `about `and `contact `links are redirecting to wrong page sections. please fix that.